### PR TITLE
Use `str_starts_with()` and `str_contains()` where possible

### PR DESCRIPTION
**Pull request by fezfez**

### CODE REVIEW

**Comment by Ocramius** [comment]
```
Error: src/Generator/ValueGenerator.php:315:46: MixedArgument: Argument 2 of str_contains cannot be mixed, expecting string (see https://psalm.dev/030)
```

Unsure if it should go to baseline or not.

File: src/Generator/ValueGenerator.php [hunk 1]
```diff
@@ -311,7 +312,7 @@ public function getAutoDeterminedType($value)
                         return self::TYPE_CONSTANT;
                     }
 
-                    if (strpos($value, $constant) !== false) {
+                    if (str_contains($value, $constant)) {
```
Comment (Ocramius) [inline src/Generator/ValueGenerator.php:315]: Yeah, probably to baseline...

Comment (fezfez) [reply src/Generator/ValueGenerator.php:315]: i'he try to find a solution but : 

```
ERROR: TooManyTemplateParams - src/Generator/ValueGenerator.php:70:13 - Laminas\Stdlib\ArrayObject<array-key, string> has too many template params, expecting 0 (see https://psalm.dev/184)
     * @var SplArrayObject<array-key, string>|StdlibArrayObject<array-key, string>
```

Comment (Ocramius) [reply src/Generator/ValueGenerator.php:315]: I think a suppression is OK here: no need to add further complexity in internal types :thinking:

Comment (fezfez) [reply src/Generator/ValueGenerator.php:315]: done

**Review by Ocramius — APPROVED** [review]
Thanks @fezfez!

**Pull request merged**